Update README.md (#86)
RMI-PACTA/workflow.data.preparation: README.md

# workflow.data.preparation

> `workflow.data.preparation` orchestrates the PACTA data preparation process, combining production, financial, scenario, and currency data into a format suitable for use in a PACTA for investors analysis. Assuming that the computing resource being used has sufficient memory (which can be >16gb depending on the inputs), storage space, and access to the necessary inputs, this is intended to work on a desktop or laptop using RStudio or in an ephemeral Docker instance created by the included [Dockerfile](https://github.com/RMI-PACTA/workflow.data.preparation/blob/main/Dockerfile) and [docker-compose.yml](https://github.com/RMI-PACTA/workflow.data.preparation/blob/main/docker-compose.yml).

Reviewer: ```suggestion
`workflow.data.preparation` orchestrates the PACTA data preparation process, combining production, financial, scenario, and currency data into a format suitable for use in a PACTA for investors analysis. Assuming that the computing resource being used has sufficient memory (which can be >16gb depending on the inputs), storage space, and access to the necessary inputs, this is intended to work on a desktop or laptop using RStudio or run using the included [Dockerfile](https://github.com/RMI-PACTA/workflow.data.preparation/blob/main/Dockerfile) and [docker-compose.yml](https://github.com/RMI-PACTA/workflow.data.preparation/blob/main/docker-compose.yml).
```

Reviewer: I don't think "ephemeral" docker instance is really well-defined. 

You "run it using Docker" in the same way you would run anything using Docker, which just means:
* build an image
* run the image, maybe with arguments

Author: hmmm... I think there's a substantial difference between building a Docker image (which implies at some point you'll expect to come back some time in the future and use it as is) and running a Docker instance that is built on the fly, does its thing, and then exits out immediately after and cleans up after itself. But, I didn't really build any of that stuff here, so I'll take your suggestion for now.
